README.md

```diff
@@ -20,6 +20,7 @@
     - No required base classes for events
  - Support for both high-level (subscription) and low-level APIs
  - Pluggable HTTP client implementations
+ - Gzip encoding for publishing and consuming events, enabled by default
```

Comment:
I am wondering if we really want to make it default. Are we sure that this won't have any side effects? Maybe we want to iterate over it like leaving it in prod for a few month and wait if there are bug reports and only if not, make it default?

Reply:
discussed and implemented differently now.